app/views/state_file/questions/az_senior_dependents/edit.html.erb

```diff
@@ -0,0 +1,46 @@
+<% content_for :page_title, "Dependents' 65+ Extra Questions" %>
+
+<% content_for :card do %>
+  <p><%= t('.title1') %></p>
+
+  <% if @form.errors.any? %>
+    <div class="notice--error">
+      <% @form.errors.full_messages.each do |msg| %>
+        <div><%= msg %></div>
+      <% end %>
+    </div>
+  <% end %>
```

Comment:
Ideally this should be handled the same way on all pages, but it's helpful to double check and affirm that we need to see the errors.